mw-webapp/src/styles/_variables.scss

```diff
@@ -0,0 +1,28 @@
+/*
+Button
+*/
+
```

Comment:
Delete empty line
And add 1 line comment instead of few line
Check it in all file

Reply:
Done.